### ISSUE TITLE
make `mutations_df` also return the predicted functional score

### ISSUE THREAD

**Issue by jbloom** [matsengrp/multidms#72]
For many analyses, it is useful to know the predicted functional scores of each single mutation. It seems like the `mutations_df` code has commented out lines that did this. I think it would be helpful to have that information returned by `mutations_df`. Note that if you do this, you also need to return the predicted functional scores for the "wildtype" mutations as wildtype is not guaranteed to have a value of zero on either the latent or predicted phenotype scale (see #73).

**Comment by jbloom** [comment]
In general, it would be great if code enabled doing something like what the hack below does for the special case of the reference condition and sigmoid global epistasis:

<img width="754" alt="Screen Shot 2023-07-10 at 5 47 22 AM" src="https://github.com/matsengrp/multidms/assets/1906801/67503814-8423-42cb-8511-af6df581b300">

**Comment by jgallowa07** [comment]
This all makes sense to me wrt individual mutation effects and I've got this implemented [in this branch](https://github.com/matsengrp/multidms/blob/78e01b91d426c8e9af1b9b2a2738901b06ed4e07/multidms/model.py#L394). 

But I have a potentially naive question, do we want also subtract the wildtype from the variant predictions (as I started to do [here](https://github.com/matsengrp/multidms/blob/78e01b91d426c8e9af1b9b2a2738901b06ed4e07/multidms/model.py#L355), and could also easily do [here](https://github.com/matsengrp/multidms/blob/78e01b91d426c8e9af1b9b2a2738901b06ed4e07/multidms/model.py#L616))?

**Comment by jbloom** [comment]
I think for the `variants_df` and `add_phenotypes_to_df` we could either report phenotypes **and** effects on phenotypes, or just the former. But I think in either case we should come up with consistent notation and add a note to the method docstrings in the API emphasizing the difference---or if we just report phenotype, note that wildtype may not be zero.

If we use the distinct names can be something like: `pred_func_score` and `predicted_latent_phenotype` and then the same with either `delta` or `effect` to indicate which we mean.

In other words, I don't have a strong preference here just we need the docs to make it obvious to reader that predicted are not the same as difference from wildtype.

**Comment by jgallowa07** [comment]
Great - thanks. To keep outputs concise, I'm just going to add a parameter to each of `mutations_df`, `variants_df`, and `add_phenotypes_to_df` which is just
```
        phenotype_as_effect : bool
            if True, phenotypes (both latent, and func_score) 
            are calculated as the _difference_ between predicted 
            phenotype of a given variant and the respective experimental 
            wildtype prediction. Otherwise, report the unmodified 
            model prediction.
```
Unless you have any objections

**Comment by jbloom** [comment]
Sounds good.
